WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift

```diff
@@ -33,11 +33,14 @@ class CommentContentTableViewCell: UITableViewCell, NibReusable {
 
     @IBOutlet private weak var webView: WKWebView!
     @IBOutlet private weak var webViewHeightConstraint: NSLayoutConstraint!
+    @IBOutlet weak var webViewBottomConstraint: NSLayoutConstraint!
```

Comment:
Should this be `private`? Since all the other `IBOutlet`s are...

Reply:
What do you think about using a stack view to display the cell contents? 
- When the reaction bar is hidden the stack view will automatically resize, negating the need to manually modify constraints. 
- It should also resolve the crash due to missing constraint.
- I think it will also facilitate showing the new moderation bar. It will be displayed conditionally in this cell (ref pbArwn-2ks#comment-3746), so having a stack view means showing/hiding it should just work.

Reply:
> Should this be private? Since all the other IBOutlets are...

Whoops! Good catch. Thanks!

> What do you think about using a stack view to display the cell contents?

Yeah, I can do that. Thanks for the suggestion!

Reply:
This is addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/17147/commits/8965bca1460c6d48119fb9d8d3e6b7c27b692fdf. I've tested the Edit Comment flow and confirmed that the crash you mentioned in https://github.com/wordpress-mobile/WordPress-iOS/pull/17147#issuecomment-918537809 no longer happens. Thanks!